app/javascript/readingList/components/ItemListItem.jsx

```diff
@@ -23,17 +23,17 @@ export const ItemListItem = ({ item, children }) => {
         />
 
         <div className="item-details">
-          <a className="item-user" href={`/${adaptedItem.user.username}`}>
+          <a datatestid="item-user" className="item-user" href={`/${adaptedItem.user.username}`}>
             <img src={adaptedItem.user.profile_image_90} alt="Profile Pic" />
             {`${adaptedItem.user.name}・`}
             {`${adaptedItem.publishedDate}・`}
             {`${adaptedItem.readingTime} min read・`}
           </a>
 
           {adaptedItem.tags.length > 0 ? (
-            <span className="item-tags">
+            <span datatestid="item-tags" className="item-tags">
               {adaptedItem.tags.map((tag) => (
-                <a className="item-tag" href={`/t/${tag.name}`}>
+                <a datatestid="item-tag" className="item-tag" href={`/t/${tag.name}`}>
```

Comment:
This `data-testid` would probably be more meaningful if it included the tag name.

```suggestion
                <a datatestid={`item-tag-${tag.name}`} className="item-tag" href={`/t/${tag.name}`}>
```

Reply:
I realised that I wasn't actually using this id 😅 and so I've removed it